### DISCUSSION TITLE
i can not scrap data from nintendo example.

### DISCUSSION THREAD

**Discussion by algarcser** [kevinzg/facebook-scraper#1099] [Q&A]
For the example provided in the code, i tried to download data but it does not return anything. i have tried scraping a post and it does work well. Is it possible that the api does not work well in another regions. I am scrapping from spain, and when i go into the facebook page for nintento i get the following url. 
https://www.facebook.com/NintendoES/?locale=es_ES
My idea is that since i get the extra ?locale=es_ES for the lenguaje it can not find it easily. I have tried scrapping NintendoES but it has also comed to no result. I wanted to see if i could scrap the information of specifics profiles and would like to figure out how to use it. I have yet to try loading a profile with which scrape data. But i wanted to know is the region could pose a problem. 

for post in get_posts('nintendo', pages=100):
    pprint(post)
    print("We have finished the scrapping")

**Comment by y-aghnaj-ux** [comment]
I think that it's due to same chnge in the html structure of facebook, since while scraping for posts, the api will look for `<article>` , while from what I noticed facebook now only uses `<div>`

**Reply by meensukt** [reply]
Does it mean this code is now obsolete?

**Reply by algarcser** [reply]
I was not able to make it work i had to use the following library https://github.com/moda20/facebook-scraper to be able to make it work. It uses this library but solves some issues. With that configured i was able to download data from Facebook again.
